openQA.spec

```diff
@@ -159,12 +159,16 @@ The openQA worker manages test engine (provided by os-autoinst package).
 %package client
 Summary:        Client tools for remote openQA management
 Group:          Development/Tools/Other
+Requires:       openQA-common = %{version}
```

Comment:
hmm, openQA-common is huge. Not a fan of this change

Reply:
We could split openQA-common. But openQA-common requires only 830.3 KiB and its biggest dependency (Mojolicious) is required anyways. So I'd say it isn't worth the effort to save a few MB.

And I'd like to note that I don't introduce the dependency here. I just update the list of dependencies.

Reply:
no one said it's your fault ;)

Reply:
'use openQA libs'? You're talking about 4745f7a38f45a1f72cb34397cd737453862e94cd, right? This means to me ./lib/OpenQA/UserAgent.pm is in the wrong package. It shouldn't be in common, but in client.

Reply:
Yes, that one